pages/common/mpc.md

```diff
@@ -1,8 +1,8 @@
 # mpc
 
 > Music Player Client.
-> Program for controlling the Music Player Daemon (MPD).
-> More information: <https://www.musicpd.org/clients/mpc>.
+> A program for controlling the Music Player Daemon (MPD).
```

Comment:
If I'm remembering correctly, we're trying to get rid of declaring that these are programs.
```suggestion
> Control the Music Player Daemon (MPD).
```